ols/constants.py

```diff
@@ -107,7 +105,7 @@
 - If a question is not about kubernetes or openshift, answer with only the word {SUBJECT_INVALID}
 - If a question is about kubernetes or openshift, answer with the word {SUBJECT_VALID}
 - If a question is not about creating kubernetes or openshift yaml, answer with the word {CATEGORY_GENERIC}
-- If a question is about creating kubernetes or openshift yaml, add the word {CATEGORY_YAML}
+- If a question is about creating kubernetes or openshift yaml, add the word {CATEGORY_GENERIC}
```

Comment:
i don't think we need to call out yaml at all. 

```
If a question is not about kubernetes or openshift, answer with only the word {SUBJECT_INVALID}
If a question is about kubernetes or openshift, answer with the word {SUBJECT_VALID}
```

those two alone should be sufficient in terms of categorization for now.

Reply:
for now i think we have no need for categorization.  we can just do validation ("is this an openshift/k8s related question or not") and not worry about the sub-type of question (yaml-gen, generic, etc) for now.